src/tools/pontoon/teams_projects.md

```diff
@@ -2,69 +2,68 @@
 
 <!-- toc -->
 
-The [Teams page](https://pontoon.mozilla.org/teams/) lists all locales that are enabled in Pontoon. From here you can access a specific [Team page](#team-page), where you’ll get a list of all projects enabled for the requested locale. Selecting a project from the *Team page* will take you to what is called the [Localization page](#localization-page).
-
-From each locale’s *Team page* it’s also possible to [request a project](#requesting-a-project), that is asking to add that locale to a project already available in Pontoon, not asking to support localization for a brand new project.
-
-The [Projects page](https://pontoon.mozilla.org/teams/) lists all projects that are available in Pontoon. From here you can access the list of locales that are enabled for the requested project. This is called a [Project page](#project-page) in Pontoon. By selecting a locale in the *Project page* you access the [Localization page](#localization-page).
-
-There is also a special Localization page that allows to access all resources for all projects enabled for a specific locale: `/locale/all-projects/all-resources`.
+Pontoon organizes localizations by locale/project pairs. Pontoon's structure reflects this with separate pages for [Teams](#teams) (locales), [Projects](#projects) and [Localizations](#localization-page).
```

Comment:
I'm not sure about this change:
* The existing page has a short explanation followed by the graph, now the graph is provided without context (the context is spread across the page later on).
* The current statement is not very clear (`Pontoon organizes localizations by locale/project pairs`)., specially when then you talk again about Localizations in the sentence after.

I think it makes sense to keep the current blurb (fixing the "you"), and accept that there will be some duplication.